t/20-stale-job-detection.t

```diff
@@ -22,19 +22,22 @@ use lib "$FindBin::Bin/lib";
 use DateTime;
 use Test::Warnings ':report_warnings';
 use Test::Output qw(combined_like stderr_like);
+use OpenQA::App;
 use OpenQA::Constants qw(DEFAULT_WORKER_TIMEOUT DB_TIMESTAMP_ACCURACY);
 use OpenQA::Jobs::Constants;
 use OpenQA::WebSockets;
 use OpenQA::Test::Database;
-use OpenQA::Test::Utils qw(setup_mojo_app_with_default_worker_timeout redirect_output);
+use OpenQA::Test::Utils qw(redirect_output);
 
 my $schema = OpenQA::Test::Database->new->create(fixtures_glob => '01-jobs.pl 02-workers.pl 06-job_dependencies.pl');
 my $jobs   = $schema->resultset('Jobs');
 $jobs->find(99963)->update({assigned_worker_id => 1});
 $jobs->find(99961)->update({assigned_worker_id => 2});
 $jobs->find(80000)->update({state              => ASSIGNED, result => NONE, assigned_worker_id => 1});
 
-setup_mojo_app_with_default_worker_timeout;
```

Comment:
When you not use this function anymore, is it still used somewhere else? If not, delete, if only once better move the function to the single place where used

Reply:
It is still used on multiple places.